RoddyCore/src/de/dkfz/roddy/client/RoddyStartupOptions.groovy

```diff
@@ -48,7 +58,9 @@ public enum RoddyStartupOptions {
     executionServiceAuth(true), 
     executionServiceHost(true), 
     executionServiceUser(true),
-    
+
+    userepository(true),
```

Comment:
Seems to be associated with some code not related to this commit. Maybe put into a new branch?